test/domains/compilerErrors/notsupportedRectOps-ok.chpl

```diff
@@ -0,0 +1,30 @@
+use notsupportedCommon;
+
+proc test(dom) {
+  writeln(dom.type:string);
+  writeln(dom.low);
+  writeln(dom.high);
+  writeln(dom.stride);
```

Comment:
Unsure: What is the stride of an associative domain?

Reply:
This is a good question! and the reason why we do not support .stride on associative domains. A sparse rectangular subdomain inherits stride from its parent.

Reply:
P.S. I tried using --ignore-errors-for-pass and I did not like it — there is too much noise in the output, and it is probably fragile w.r.t. compiler/module changes. So I tested just the key representatives.